taxi/views.py

```diff
@@ -1,3 +1,19 @@
+from lib2to3.fixes.fix_input import context
```

Comment:
It seems like the import from `lib2to3.fixes.fix_input` is not being used in this script. It's a good practice to remove unused imports to keep the code clean and readable.